### ISSUE TITLE
onDurationChanged not called on Flutter Web

### ISSUE THREAD

**Issue by sc00n** [bluefireteam/audioplayers#878]
When I use 

```
 _durationSubscription = _audioPlayer.onDurationChanged.listen((duration) {
      setState(() => _duration = duration);
      ...
      });
```

_duration is never set in Flutter web. Therefor the progress bar also never changes. It works fine on iOS and Android.

**Comment by erickzanardo** [comment]
Please check our feature_parity_table.md file to check the features available on each platform.

Events are not available on web yet

**Comment by Gustl22** [comment]
Fixed via #1268
